style.css

```diff
@@ -1,6 +1,11 @@
 body {
   font-family: Arial, sans-serif;
   background-color: #f2f2f2;
+  background: url("https://png.pngtree.com/thumb_back/fh260/background/20201009/pngtree-clear-blue-sky-with-white-clouds-panoramic-view-of-bright-clear-image_403142.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100vw;
+  height: 100vh;
```

Comment:
these values are introducing scroll, pls make sure you update these values so that no scrolling is introduced